packages/manager/apps/web/client/app/hosting/multisite/hosting-multisite.controller.js

```diff
@@ -91,6 +89,10 @@ angular
         value: period,
       }));
 
+      HostingDomain.getZones().then((zones) => {
+        $scope.listZone = zones;
+      });
```

Comment:
no need to manage potential error here ?

Reply:
I added the error case just an empty array because no need error message.